### ISSUE TITLE
2021-12-12

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#682]
151python train.py train multi_resnet18_kd --data-dir /root/pytorch-be-your-own-teacher/cifar-100-python
152Test: 
153python train.py test multi_resnet18_kd  --data-dir /root/pytorch-be-your-own-teacher/cifar-100-python 
154python train.py test multi_resnet18_kd  --data-dir /root/pytorch-be-your-own-teacher/cifar-100-python/cifar-100-python
155python train.py test multi_resnet18_kd --data-dir root/pytorch-be-your-own-teacher/cifar-100-python --resume /root/pytorch-be-your-own-teacher/save_checkpoints
156Draw learning rate curve of decay 1, to decide at which to stop; 
157In bioinformatics, a sequence alignment is a way of arranging the sequences of DNA, RNA, or protein to identify regions of similarity that may be a consequence of functional, structural, or evolutionary relationships between the sequences.[1] Aligned sequences of nucleotide or amino acid residues are typically represented as rows within a matrix. Gaps are inserted between the residues so that identical or similar characters are aligned in successive columns. Sequence alignments are also used for non-biological sequences, such as calculating the distance cost between strings in a natural language or in financial data. (2021-12-1)
158Ask signature for display; done; carry it to lab; done; 24-inch is enough; (2021-12-6)
159softmax归一化; (2021-12-7)
